lib/config.js

```diff
@@ -91,7 +91,9 @@ Config.prototype.buildArgs = function () {
     chrome_version_major: chrome_version_parts[0],
     safebrowsing_api_endpoint: this.safeBrowsingApiEndpoint,
     brave_referrals_api_key: this.braveReferralsApiKey,
-    enable_hangout_services_extension: this.enable_hangout_services_extension
+    enable_hangout_services_extension: this.enable_hangout_services_extension,
+    enable_printing: false,  // disable cloudprint
```

Comment:
is this just cloud printing? cc @darkdh

Reply:
Whoops, that's all of printing. I found the flag at https://github.com/Eloston/ungoogled-chromium/blob/master/patches/inox-patchset/0005-disable-default-extensions.patch#L19-L23. But it looks like these particular lines need to be removed in order to disable just cloud printing.